phoenicis-javafx/src/main/java/org/phoenicis/javafx/dialogs/ConfirmDialog.java

```diff
@@ -4,16 +4,13 @@
 import javafx.beans.property.SimpleObjectProperty;
 import javafx.scene.control.Alert;
 import javafx.scene.control.ButtonType;
-import javafx.scene.control.Label;
-import javafx.scene.layout.Region;
-import javafx.stage.Window;
 
 import java.util.Optional;
 
 /**
- * A confirm dialog with two callbacks
+ * An abstract class for confirm dialogs with two callbacks
  */
-public class ConfirmDialog extends Alert {
+public abstract class ConfirmDialog extends Alert {
```

Comment:
Wouldn't it make sense to already have title, owner etc in this class?

Reply:
This information is part of `Alert`.

Reply:
Ah I messed up with the builder.